upgrade/1.2/scripts/common/ncn-rebuild-common.sh

```diff
@@ -48,36 +48,6 @@ else
     echo "====> ${state_name} has been completed"
 fi
 
-state_name="ELIMINATE_NTP_CLOCK_SKEW"
```

Comment:
Would we not want to also check the sync here? I know we check it at the prereq step, but in some upgrades there may be days between when the prereq script is run and any given node is rebuilt. It seems long enough that we'd want to verify the clocks have not drifted, right?

Reply:
Yeah, I can do the `makestep` here and also check.  We shouldn't have to re-run csm_ntp.py though.